### DISCUSSION TITLE
How to list all users?

### DISCUSSION THREAD

**Discussion by MKE1138** [jellyfin/jellyfin#5176] [Q&A] [answered]
I created an administrator account and can't remember the username. Is there any way to find al list of all user accounts?

**Comment by anthonylavado** [comment, marked as answer]
The answer depends on the operating system/install method you're using.

For Linux, there are user folders at `/etc/jellyfin/users/`.

**Reply by MKE1138** [reply]
Ah, cool. I’m running it in a container, so I’m guessing the users are at listed at the same path in the container?

**Reply by anthonylavado** [reply]
Yeah, that should be the case 🙂

**Comment by SupinePandora43** [comment]
Doesn't work

**Reply by MKE1138** [reply]
Seeing how this was from 4 years ago, that's not surprising
